ui/v2.5/src/components/Settings/SettingsTasksPanel/SettingsTasksPanel.tsx

```diff
@@ -91,8 +93,11 @@ export const SettingsTasksPanel: React.FC = () => {
     if (jobStatus?.data?.jobStatus) {
       setStatus(statusToText(jobStatus.data.jobStatus.status));
       const newProgress = jobStatus.data.jobStatus.progress;
+      console.log('jobStatus:', jobStatus);
```

Comment:
Can't seem to get the `started` variable here

Reply:
You need to add the started field to `graphql/documents/queries/settings/metadata.graphql`. All frontend queries are defined in the `graphql/documents` folder. When you then run `make generate`, or `yarn gqlgen` in the ui folder, this populates the types.